packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -33,8 +33,9 @@ class TextPainter {
   TextPainter({
     TextSpan text,
     TextAlign textAlign,
-    double textScaleFactor: 1.0
-  }) : _text = text, _textAlign = textAlign, _textScaleFactor = textScaleFactor {
+    double textScaleFactor: 1.0,
+    String ellipsis
```

Comment:
nit: these days, when you have a list like this, stick a trailing comma in there so future changes don't need to edit the previous line